### DISCUSSION TITLE
Cant install OKD 4.9 vmware UPI

### DISCUSSION THREAD

**Discussion by dkoci** [okd-project/okd#1067] [Q&A]
Hi.
I cant install OKD 4.9 (vmware, UPI). **I have set hostnames in DHCP**. There is an error on bootstrap node.

![image](https://user-images.githubusercontent.com/5565897/149370161-f6222bbd-fe80-49bd-afbb-add7f2110849.png)

**Comment by rvanderp3** [comment]
Can you provide logs from the installation attempt? 

https://docs.okd.io/latest/installing/installing-troubleshooting.html#installation-bootstrap-gather_installing-troubleshooting

**Reply by dkoci** [reply]
I tried again. Bootstrap does not finish. I attached gather logs: https://www.filemail.com/d/sncigpmzdcwkyjw

**Reply by rvanderp3** [reply]
From looking over the logs, it appears that the bootstrap is progressing, but could be waiting for the control plane nodes.  Are there any errors on the control plane node consoles?  Their logs didn't seem to get picked up.

**Reply by dkoci** [reply]
I cant ssh to control plane nodes (ssh core@<node_name>) gives me an error: ` Permission denied (publickey,gssapi-keyex,gssapi-with-mic).`

Guest os screen of master01 node in vmware:

![image](https://user-images.githubusercontent.com/5565897/149387417-a1fda685-707c-4fd2-bf6a-31d40c114913.png)

**Reply by rvanderp3** [reply]
It sounds like the control plane nodes haven't finished igniting and applying the ssh key.  They cleared the first stage of ignition which is consistent with what was seen in the logs.  I'm curious if you see network activity on the control plane nodes in the VMware console.  It may just be that the nodes are taking longer than normal to install.

**Reply by dkoci** [reply]
But so long? More than 2 hours? Is there anything else I can try?

**Reply by rvanderp3** [reply]
2 hours is a long time for sure.  You might try adding a serial port on one of the control plane nodes and have it write the output to a file.  We might be able to determine how far along the node has gotten based on that information.

**Reply by dkoci** [reply]
Can you please give me some tips how to add serial port?

**Reply by dkoci** [reply]
I finally made it. I tried to reinstall it from scratch. I regenerated ssh keys. I also replaced fcos OVA template from v35 (hardware version 13) to v34 (hardware version 15). The cluster is successfully installed.

**Reply by rvanderp3** [reply]
@dkoci I'm glad you were able to get that installed.  I wonder what the problem could have been.

**Reply by rarguello** [reply]
Maybe this https://github.com/openshift/okd/issues/1020
